### PR TITLE
Feature/startup interstitial

### PR DESCRIPTION
**Pull request by gfodor**
This PR:

- Adds an easy to skip interstitial on first launch with blank local storage to prompt the user to sign up for our release notes list.

- Adds the ToU and PP to the two main entry flows.

### CODE REVIEW

File: src/react-components/info-dialog.js [hunk 1]
```diff
@@ -140,14 +156,20 @@ class InfoDialog extends Component {
...
-                <label className="mailing-list-form__privacy">
+                <label
+                  className={classNames({
+                    "mailing-list-form__privacy": true,
+                    "mailing-list-form__privacy--hidden":
+                      this.state.mailingListEmail.length === 0 && this.props.allowEmailSkip
+                  })}
+                >
                   <input
                     className="mailing-list-form__privacy_checkbox"
```
Comment (brianpeiris) [inline src/react-components/info-dialog.js]: Maybe use the `%checkbox` styling, for consistency with the rest of the UI? Might have to scale it down to match the text though.

File: src/react-components/home-root.js [hunk 2]
```diff
@@ -154,6 +154,22 @@ class HomeRoot extends Component {
                   >
                     <FormattedMessage id="home.report_issue" />
                   </a>
+                  <a
+                    className="footer-content__links__link"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href="https://github.com/mozilla/hubs/blob/master/TERMS.md"
```
Comment (brianpeiris) [inline src/react-components/home-root.js]: Are we actually just going to link to github for the notices? We ought to be able to generate some styled HTML from the markdown at least.

**Comment by brianpeiris** [comment]
The screen looks very dark and void-like with the interstitial up. Maybe a white/grey border around the dialog would help?

image

File: src/react-components/home-root.js (hunk 2 above)
Comment (gfodor) [reply src/react-components/home-root.js]: Yeah this is what was suggested by legal -- not 100% sure why but seems good to have a ground truth for now

**Comment by gfodor** [comment]
Abandoned in favor of more user explicit choice to share email.

**Pull request closed without merging**